### PR TITLE
Add release note about new LCB VM in 9.0-dp-2

### PR DESCRIPTION
**Pull request by livecodefraser**

### CODE REVIEW

File: docs/notes/new-lcb-vm.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Re-written LCB VM
+
+The "virtual machine" used to run LiveCode Builder code has been re-written
+from scratch. This new VM provides a framework enabling better extensibility,
+better error reporting and, in future, more comprehensive optimizations.
+
+Most exiting LCB code should run without any changes. There may be some code
```
Comment (peter-b) [inline docs/notes/new-lcb-vm.md]: "exiting" -> "existing"

**Comment by peter-b** [comment]
@livecode-vulcan review ok 8cacd61

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 8cacd617848a79d37da7135e7e75eb40fc7f4356

**Comment by livecode-vulcan** [comment]
😎 test success 8cacd617848a79d37da7135e7e75eb40fc7f4356
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/620)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/625)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/620)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/624)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/629)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/621)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/621)

**Pull request merged**